core/templates/domain/improvements/task-entry.model.ts

```diff
@@ -34,11 +34,11 @@ export interface TaskEntryBackendDict<TaskType = string> {
   'task_type': TaskType;
   'target_type': string;
   'target_id': string;
-  'issue_description': string;
+  'issue_description': string | null;
```

Comment:
Please use `?` instead of ` | null`, here and elsewhere.

Reply:
But @vojtechjelinek told to use `| null` instead of `?`

Reply:
Ah gotcha, that's fine with me